src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx

```diff
@@ -60,12 +118,17 @@ export class ShowHeader extends React.Component<Props> {
             onViewAllPressed={onViewAllArtistsPressed}
           />
           <Spacer mt={1} />
-          <InvertedButton
-            text="Save show"
-            onPress={() => {
-              onSaveShowPressed()
-            }}
-          />
+          <ButtonWrapper>
+            <Spacer m={2} mt={1} />
+            <InvertedButton
+              inProgress={isFollowedChanging}
+              text={is_followed ? "Show saved" : "Save show"}
+              selected={is_followed}
+              onPress={this.handleFollowShow}
+              grayBorder={true}
+            />
+            <Spacer m={1} />
```

Comment:
for vertical spacing can do `my={1}` (and if you need only x spacing can do `mx`)